src/ui.js

```diff
@@ -0,0 +1,157 @@
+    div4list.appendChild(list);
+
+    appDiv.appendChild(div4list);
+  }
+
+  static addEmptyToDoMessage() {
+    const list = document.querySelector('#task-list');
+
+    const item = document.createElement('LI'); // creates list item
+    item.className = 'd-flex justify-content-center align-items-center border-bottom border-2 px-2 appItem';
+
+    const p = document.createElement('P');
+    p.className = 'm-0 p-0 noToDos';
+    p.innerHTML = "No to-do's right now";
+
+    item.appendChild(p);
+    list.appendChild(item);
+  }
+
+  static addTasksUI(tasks) {
+    // Iterates over array tasks to populate HTML list
+    if (tasks.length === 0) {
+      this.addEmptyToDoMessage();
+    } else {
+      tasks.forEach((task) => this.addTaskToList(task));
+    }
+  }
+
+  static addTaskToList(task) {
+    const list = document.querySelector('#task-list');
+
+    const item = document.createElement('LI'); // creates list item
+    item.className = 'd-flex justify-content-around align-items-center border-bottom border-2 px-2 appItem';
+
+    // creates div for normal view
+    const divNormal = document.createElement('DIV');
+    divNormal.className = 'd-flex align-items-center normalView';
+
+    const checkbox = document.createElement('INPUT'); // creates checkbox
+    checkbox.setAttribute('type', 'checkbox');
+    checkbox.checked = task.completed;
+    checkbox.className = 'form-check-label p-2';
+    divNormal.appendChild(checkbox); // appends checkbox to item
+
+    const text = document.createElement('P'); // creates p
+    text.textContent = task.description;
+    text.className = 'm-0 p-2';
+    if (task.completed === true) {
+      text.classList.add('text-decoration-line-through');
+    }
+    divNormal.appendChild(text); // appends p to item
+
+    // Creates div for icons
+    const div4Icons = document.createElement('DIV');
+    div4Icons.className = 'ms-auto';
+
+    const iconEdit = document.createElement('I'); // creates edit icon
+    iconEdit.className = 'fas fa-ellipsis-v p-2 edtIcn';
+    div4Icons.appendChild(iconEdit); // appends edit icon to item
+
+    const iconAccept = document.createElement('I'); // creates accept icon
+    iconAccept.className = 'fas fa-check-circle p-2 d-none acceptIcn';
+    div4Icons.appendChild(iconAccept); // appends accpet icon to item
+
+    const iconRemove = document.createElement('I'); // creates remove icon
+    iconRemove.className = 'fas fa-trash p-2 d-none removeIcn';
+    div4Icons.appendChild(iconRemove); // appends remove icon to item
+
+    item.appendChild(divNormal);
+    // item.appendChild(divEdit);
+    item.appendChild(div4Icons);
+
+    list.appendChild(item); // appends item to list
+  }
+
+  static addbottombtn() {
+    const appDiv = document.querySelector('#appDiv');
+
+    const btmDiv = document.createElement('DIV');
+    btmDiv.className = 'd-flex justify-content-center align-items-center border btmDiv';
+
+    const pBtm = document.createElement('P');
```

Comment:
- [ ]  Let's follow the HTML best practices by using lowercase for elements and attribute names. Check the comment above for more details.